WHITEPAPER.md

```diff
@@ -135,6 +135,8 @@ total amount of tokens held by each zone.  The hub isolates each zone from the
 failure of other zones.  Because anyone can connect a new zone to the Cosmos Hub,
 zones allow for future-compatibility with new blockchain innovations.
 
+With Cosmos interoperability between blockchains can be achieved. The potential of an internet of value, where assets are issued and controlled by different sets of validators, yet can be moved and exchanged seamlessly between blockchains without relying on trusted third parties can be realized.
```

Comment:
`The potential of an Internet of Value can be realized, where assets are operated by different validation schemes, yet can be moved and exchanged seamlessly between blockchains without relying on trusted third parties.`